src/settings/Settings.tsx

```diff
@@ -698,6 +698,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td('Send analytics data'),
         default: null,
     },
+    "ftueUseCaseSelection": {
+        supportedLevels: [SettingLevel.ACCOUNT],
```

Comment:
so it can be overridden safely:
```suggestion
        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
```

Reply:
this will also be the way for managed deployments to disable it.